### PR TITLE
[env/burn-staging] Fix events in schedule and fix chats header

### PR DESCRIPTION
**Pull request by ermiaSparkle**
Resolve issue: 
- https://github.com/sparkletown/internal-sparkle-issues/issues/1002

### CODE REVIEW

File: src/components/molecules/NavBar/NavBar.scss [hunk 1]
```diff
@@ -165,7 +165,7 @@ $volume-popover-width-without-sliders: 15em;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  top: $navbar-height;
+  top: $navbar-height + 50px;
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.scss:168]: If you add plain values, it is hard to figure out why exactly that `50px` and not `42px`. There are constants just like `$navbar-height` that will inform development of the reason for their use. This one can be as well.

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 2]
```diff
@@ -145,6 +145,7 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
                   className={venueChatTabStyles}
                   onClick={selectVenueChat}
                 >
+                  <FontAwesomeIcon icon={faCommentDots} size="sm" />
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:148]: `ButtonNG` supports icons with more concise syntax. Example
 ```tsx
        <ButtonNG
          isLink
          className="RunTabView__advanced"
          linkTo={adminNGSettingsUrl(venueId)}
          iconName={faCog}
        >
          Advanced Settings
        </ButtonNG>
```

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 3]
```diff
@@ -155,6 +156,7 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
                   className={privateChatTabStyles}
                   onClick={selectPrivateChat}
                 >
+                  <FontAwesomeIcon icon={faEnvelope} size="sm" />
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:159]: `ButtonNG` supports icons

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 4]
```diff
@@ -170,10 +158,16 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
+          ? eventsFilledWithPriority.filter(
+              (event) =>
+                event.isSaved &&
+                event.venueId === currentVenue?.id?.toLowerCase()
+            )
+          : eventsFilledWithPriority.filter((event) => event.isSaved)
         : filterRelatedEvents
         ? eventsFilledWithPriority.filter(
-            (event) => event.room === venueRoomTitle
+            (event) => event.venueId === currentVenue?.id?.toLowerCase()
```
Comment (goran-peoski-work) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:170]: Hard to follow which expressions are which operands of the multiple `?:` operators. Recommend extracting the values into manageable variables before `return`

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Some changes requested for clarity of code and ease of maintenance

File: src/components/molecules/NavBar/NavBar.scss [hunk 5]
```diff
@@ -9,6 +9,7 @@ $menu-item-size: $spacing--lg * 2;
 $logo-size: $spacing--xxl + $spacing--md;
 $volume-popover-width-with-sliders: 30em;
 $volume-popover-width-without-sliders: 15em;
+$spaceBackButton: 50px;
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.scss:12]: Before adding new variables/constants/any named elements,  surrounding ones can help you keep with the ad hoc convention:
```suggestion
$back-button-space: 50px;
```

In the above example:

- No camelCase
- No noun at first position 
  - logo size <- adjective noun
  - space back button <- is it the button or is it the space that is representing the value?

File: src/components/molecules/NavBar/NavBar.scss [hunk 6]
```diff
@@ -191,7 +192,7 @@ $simplified-view-button--padding: 0 10px;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  top: $navbar-height;
+  top: $navbar-height + $spaceBackButton;
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.scss:195]:
```suggestion
  top: $navbar-height + $back-button-space;
```

File: src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx [hunk 7]
```diff
@@ -34,15 +34,15 @@ export const ScheduleVenueDescription: FC<ScheduleVenueDescriptionProps> = ({
...
+    currentVenue?.config?.landingPageConfig ?? {};
 
   return (
     <div className={containerClasses}>
       <div className="ScheduleVenueDescription__main">
         <div className="ScheduleVenueDescription__pic" />
         <div className="ScheduleVenueDescription__title">
           <h2 className="ScheduleVenueDescription__name">
-            {sovereignVenue?.name ?? "Schedule"}
+            {currentVenue?.name ?? "Schedule"}
```
Comment (goran-peoski-work) [inline src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx:45]: Maybe extract a constant
```
const DEFAULT_SCHEDULE_NAME = "Schedule";
```

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 8]
```diff
@@ -167,14 +155,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       eventTimeAndOrderComparator
     );
 
+    const currentVenueBookMarkEvents = eventsFilledWithPriority.filter(
+      (event) =>
+        event.isSaved && event.venueId === currentVenue?.id?.toLowerCase()
```
Comment (goran-peoski-work) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:160]: If possible
```
const currentVenueId = currentVenue?.id?.toLowerCase()
```
before running it multiple times inside the filter with both sides of the `===` lower-cased

```suggestion
      ({ isSaved, venueId }) =>
        isSaved && venueId?.toLowerCase() === currentVenueId
```

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 9]
```diff
@@ -167,14 +155,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
+    );
+
+    const currentVenueEvents = eventsFilledWithPriority.filter(
+      (event) => event.venueId === currentVenue?.id?.toLowerCase()
```
Comment (goran-peoski-work) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:164]: `currentVenueId` constant like above

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Few minor code change suggestions and be good to go

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx (hunk 2 above)
Comment (ermiaSparkle) [reply src/components/organisms/ChatSidebar/ChatSidebar.tsx]: fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx (hunk 3 above)
Comment (ermiaSparkle) [reply src/components/organisms/ChatSidebar/ChatSidebar.tsx]: fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx (hunk 4 above)
Comment (ermiaSparkle) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54

File: src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx (hunk 7 above)
Comment (ermiaSparkle) [reply src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx]: fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx (hunk 9 above)
Comment (ermiaSparkle) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 10]
```diff
@@ -167,14 +155,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
     );
 
+    const currentVenueBookMarkEvents = eventsFilledWithPriority.filter(
+      ({ isSaved, venueId: eventVenueId }) =>
+        isSaved && eventVenueId?.toLowerCase() === venueId
+    );
+
+    const currentVenueEvents = eventsFilledWithPriority.filter(
+      (event) => event.venueId === venueId?.toLowerCase()
+    );
```
Comment (goran-peoski-work) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:165]: This is inconsistent. 

The original suggestion was to have a `currentVenueId` constant outside that has already been updated to lower case as to not repeat the operation and just use `venueId.toLowerCase()` inside so they're both reduced to lower before comparison

Now the namings are off, one filter function uses destructuring, the other doesn't, and there's unnecessary renaming venueId -> eventVenueId

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Filter functions inconsistency

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx (hunk 10 above)
Comment (ermiaSparkle) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: fix in c85b1f92ed600fc632d74bac6c9a9afb99f16203

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx (hunk 9 above)
Comment (ermiaSparkle) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: fix in c85b1f92ed600fc632d74bac6c9a9afb99f16203

File: src/components/organisms/ChatSidebar/ChatSidebar.scss [hunk 11]
```diff
@@ -151,7 +160,7 @@ $help-center-background-color: #232225;
     width: 60%;
   }
 
-  @media only screen and (min-device-width: $media-width--mobile-min) and (max-device-width: $media-width--mobile-max) {
+  @media only screen and (max-device-width: $media-width--mobile-min) {
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.scss:163]: Good catch 👍

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 12]
```diff
@@ -20,15 +20,14 @@ import {
...
-import { getLastUrlParam, getUrlWithoutTrailingSlash } from "utils/url";
 
 import { useRelatedVenues } from "hooks/useRelatedVenues";
 import { useRoomRecentUsersList } from "hooks/useRoomRecentUsersList";
 import { useShowHide } from "hooks/useShowHide";
 import { useUser } from "hooks/useUser";
 import useVenueScheduleEvents from "hooks/useVenueScheduleEvents";
 
-import { NavBarScheduleWeather } from "components/molecules/NavBarScheduleWeather";
+// import { NavBarScheduleWeather } from "components/molecules/NavBarScheduleWeather";
```
Comment (goran-peoski-work) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: Comments in the import section can cause issues with the automatic sorting plugin

```suggestion
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 13]
```diff
@@ -245,10 +245,13 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
   });
 
+  const selectedVenue =
+    (filterRelatedEvents ? venueId : sovereignVenue?.id) ?? "";
+
   return (
     <div className="NavBarWrapper">
       <div className={containerClasses}>
-        <NavBarScheduleWeather containerClassName="NavBarSchedule--end-to-end" />
+        {/* <NavBarScheduleWeather containerClassName="NavBarSchedule--end-to-end" /> */}
```
Comment (sunny-viktoryia) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:254]: why is that commented? @ermiaSparkle

Comment (whoaitsaimz) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:254]: Looks like we are trying to solve too many things in one PR.

Comment (ermiaSparkle) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:254]: In our morning session, we decided to comment on the weather until further notice.
(CC @sofisparkle )

Comment (sunny-viktoryia) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:254]: i see, thanks! i thought it was removed by mistake because there is no reference to that issue/request in your PR.
